packages/snaps-sdk/src/jsx/components/form/AddressInput.ts

```diff
@@ -8,6 +8,7 @@ export type AddressInputProps = {
   chainId: CaipChainId;
   placeholder?: string | undefined;
   disabled?: boolean | undefined;
+  displayName?: string | undefined;
```

Comment:
I think displayAvatar makes more sense here 🤔

Reply:
At first I had it as that, but it sounds like a variable name for a boolean rather than a string. Also avatar makes me think avatar and not a name. The avatar is displayed because we have a name 😄

Reply:
Oh, sorry!

The intention isn't that the Snap can provide a name, but simply that it provides a boolean to indicate that an address is valid enough to show an avatar

Reply:
Yeah, but we don't have a name to display then?

Reply:
So the idea would be to flip it to true if the address passes validation, e.g. `isValidChecksummedAddress` or whatever

Reply:
https://www.figma.com/design/gHWl7G8ih3YXesly2XaA0R/Snaps-DS?node-id=1492-372&t=u33dSlNN0vxD951X-0

Reply:
Ah I see, I mean if the snap is indicating a match, why not let it suggest a name haha?

Reply:
We can have both props

Reply:
IMO add `displayAvatar` for now and we can always iterate to add more

Reply:
Just to make it as easy as possible to get this done